app/data_model/session_store.py

```diff
@@ -76,9 +76,9 @@ def _load(self):
             self.user_id = self._eq_session.user_id
 
             if self._eq_session.session_data:
-                session_data = self._eq_session.session_data
-                if self._is_session_data_encrypted(session_data):
```

Comment:
`_is_session_data_encrypted(session_data)` is also no longer needed. Somehow it's made it's way back in.

Reply:
@dwyeradam looking at my branch on Github it isn't there?
https://github.com/ONSdigital/eq-survey-runner/blob/eq-1812-remove-redundant-code/app/data_model/session_store.py#L78

Reply:
It's not that line that's the issue. It's the fact you still have the method in that the line was previously calling but is now no longer needed as nothing else calls it. 
https://github.com/ONSdigital/eq-survey-runner/blob/eq-1812-remove-redundant-code/app/data_model/session_store.py#L94